Package.swift

```diff
@@ -16,14 +16,18 @@ let package = Package(
             targets: ["ScipioKit"]),
     ],
     dependencies: [
+        // A revision after Xcodeproj module is removed
+        // https://github.com/apple/swift-package-manager/pull/5748
         .package(url: "https://github.com/apple/swift-package-manager.git",
-                 revision: "swift-5.7-RELEASE"),
```

Comment:
swift-package-manager on `swift-5.7-RELEASE` contains `Xcodeproj` module.
Unfortunately, its conflicts with `tuist/XcodeProj`.

So I pined a version to the latest.

@freddi-kit